### DISCUSSION TITLE
Missing baseOGImage fails website build

### DISCUSSION THREAD

**Discussion by thamerla** [pacollins/calligraphy#12] [General]
So, to reference #10 - I also came across a similar issue and it looks like it's caused by missing baseOGImage, which by default is [a screenshot of the website](https://github.com/pacollins/calligraphy/blob/main/exampleSite/assets/img/screenshot.jpg)
Was it a behavior that you wanted to implement? What was your goal behind the baseOGImage variable?

**Comment by pacollins** [comment]
Hey!

`baseOGImage` is the default OpenGraph image (picture when sharing on Twitter, Facebook, Etc) for the site. So if a page doesn't have an image for some reason, there will still be a fallback image. The way it is currently set up it is required to have this image because its a feature that most people will ultimately want to use, though I am open to changing it.

Hope that helps!

**Reply by thamerla** [reply]
Hey!

Thanks for the excellent explanation. It's clear now.
